src/client/components/MealDetails.js

```diff
@@ -0,0 +1,121 @@
+import React, { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import AddReservation from './AddReservation';
+import AddReview from './AddReview';
+
+const MealDetails = () => {
+    //useParams to get the id value for meal
```

Comment:
It's not clear what MealDetails does. Try to separate its content a bit more here

Reply:
Thanks Niels for the feedback..
i tried to make it more components and changed the name also